packages/csv-parser-discount-code/src/cli.js

```diff
@@ -0,0 +1,157 @@
+import fs from 'fs'
+import tmp from 'tmp'
+import npmlog from 'npmlog'
+import PrettyError from 'pretty-error'
+import yargs from 'yargs'
+
+import CsvParser from './main'
+import { version } from '../package.json'
+
+process.title = 'csvParserDiscountCode'
+
+const args = yargs
+  .usage(`
+Usage: $0 [options]
+Convert commercetools discount codes CSV data to JSON.`,
+  )
+  .showHelpOnFail(false, 'Use --help to display the CLI options.\n')
+
+  .option('help', {
+    alias: 'h',
+  })
+  .help('help', 'Show help text.')
+
+  .option('version', {
+    alias: 'v',
+    type: 'boolean',
+  })
+  .version('version', 'Show version number.', version)
```

Comment:
we can just refactor this to `.version()` and yargs will automatically fetch the version number for us from `package.json`